src/ResourceManager/HDInsight/ChangeLog.md

```diff
@@ -1,4 +1,4 @@
-<!--
```

Comment:
@momusoru I merged a PR yesterday (https://github.com/Azure/azure-powershell/pull/3634) that fixes an encoding issue with the markdown files, so this change needs to be reverted (you can pull in the latest from dev to get this change) to ensure that this template header is hidden when viewing the markdown on GitHub